LadybugTools_Engine/Python/src/ladybugtools_toolkit/plot/_utci.py

```diff
@@ -854,6 +919,147 @@ def utci_histogram(
                 fontsize="small",
             )
 
-    ax.yaxis.set_major_formatter(PercentFormatter(xmax=1, decimals=2))
+    ax.yaxis.set_major_formatter(mticker.PercentFormatter(xmax=1, decimals=2))
 
     return ax
+
+
+def utci_shade_benefit(
```

Comment:
The formatting of this plot method seems incomplete, note the colourbars and the ticks along the colourbars, it seems that there is an extra colour defined somewhere and included in the colourbar for some reason:
(comparing an epw file with openfield and fully enclosed)
![image](https://github.com/user-attachments/assets/ac76dfea-2e71-4dea-a275-8d5cdc0bb480)
![image](https://github.com/user-attachments/assets/3ac0dd6c-7f41-475d-ae22-fa7b6aa377ea)